src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -25,7 +25,31 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
 
         hdf5storage = get_package(package_name="hdf5storage")
 
-        super().__init__(spikes_matfile_path=file_path, verbose=verbose)
+        file_path = Path(file_path)
+
+        # Temporary hack to get sampling frequency from the spikes cellinfo file until next SI release.
+        is_new_cell_explorer_format = "spikes.cellinfo.mat" in file_path.name
+        sampling_frequency = None
+
+        if is_new_cell_explorer_format:
+            import h5py
+
+            try:
+                matlab_file = scipy.io.loadmat(file_name=str(file_path), simplify_cells=True)
+                spikes_mat = matlab_file["spikes"]
+                assert isinstance(spikes_mat, dict), f"field `spikes` must be a dict, not {type(spikes_mat)}!"
+
+            except NotImplementedError:
+                matlab_file = h5py.File(name=file_path, mode="r")
+                spikes_mat = matlab_file["spikes"]
+                assert isinstance(spikes_mat, h5py.Group), f"field `spikes` must be a Group, not {type(spikes_mat)}!"
+
+            sampling_frequency = spikes_mat.get("sr", None)
+            sampling_frequency = (
+                sampling_frequency[()] if isinstance(sampling_frequency, h5py.Dataset) else sampling_frequency
+            )
```

Comment:
Just wondering - is this guaranteed to always cast properly as a `np.float64`?

Reply:
No, this extracts the data the h5py dataset no matter the nesting.

Reply:
Doesn't SI need sampling frequency attribute to be of specific data type? I'm just saying a simple `float` cast never hurt anyone 😅

Reply:
A simple float cast will not work if the sampling_frequency is of type Dataset as far as I remember.

Reply:
Gotcha, OK then